test/fixtures_public.js

```diff
@@ -7,6 +7,7 @@
 // 05059996666: registered user - mother that receives own messages, linked to 05059997777
 // 05059997777: registered user - family member that receives household messages for 05059996666
 // 05059998888: number being changed to
+// 05059999999: number used to test sms app; does unsubscribe
```

Comment:
Ah, sorry I've done this in an issue that hasn't landed yet.  Please look in #55 if you can find an applicable piece of code to use.